packages/marathon/build

```diff
@@ -24,20 +35,21 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/marathon
 EnvironmentFile=-/opt/mesosphere/etc/marathon-extras
 EnvironmentFile=-/var/lib/dcos/marathon/environment.ip.marathon
+Environment=JAVA_HOME=${JAVA_HOME}
 ExecStartPre=/bin/ping -c1 leader.mesos
 ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-marathon
 ExecStartPre=/bin/bash -c 'echo "HOST_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" > /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "MARATHON_HOSTNAME=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
 ExecStartPre=/bin/bash -c 'echo "LIBPROCESS_IP=\$(\$MESOS_IP_DISCOVERY_COMMAND)" >> /var/lib/dcos/marathon/environment.ip.marathon'
-ExecStart=/opt/mesosphere/bin/java \\
-    -Xmx2G \\
-    -jar "$PKG_PATH/usr/marathon.jar" \\
-    --zk "\$MARATHON_ZK" \\
+ExecStart=$PKG_PATH/marathon/bin/marathon \\
+    -Duser.dir=/var/lib/dcos/marathon \\
+    -J-server \\
+    -J-verbose:gc \\
+    -J-XX:+PrintGCDetails \\
+    -J-XX:+PrintGCTimeStamps \\
     --master zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181/mesos \\
-    --hostname "\$MARATHON_HOSTNAME" \\
     --default_accepted_resource_roles "*" \\
     --mesos_role "slave_public" \\
-    --max_tasks_per_offer 100 \\
```

Comment:
`--hostname` was removed because `$MARATHON_HOSTNAME` is implicitly passed as that parameter, and it was redundant.

`--max_instances_per_offer` has been fixed